### DISCUSSION TITLE
Test if x-ref is visible?

### DISCUSSION THREAD

**Discussion by yabdab** [alpinejs/alpine#3936] [1. Help]
Does Alpine provide a way to see if an x-ref is visible ?

Like was it hidden/shown using x-show?

And it need to use bind, so it automatically lets app know if it was hidden.

Thanks

**Comment by ekwoka** [comment]
This might be an XY problem.

What is the reason you'd need this information? The best way can differ based on the true goal.

**Reply by yabdab** [reply]
I have some form inputs that I want to hide conditionally. If hidden , then do not send to server fro processing.

However, I  created a different approach and will likely go that route.

**Reply by ekwoka** [reply]
Just set them as hidden on the same condition.
